bin/test-dag-tags.py

```diff
@@ -0,0 +1,110 @@
+#!/bin/python3
+from collections import defaultdict
+import subprocess
+from typing import Dict
+
+
+VALID_TAGS = ("impact/tier_1", "impact/tier_2", "impact/tier_3", "repo/bigquery-etl", "repo/telemetry-airflow",)
+REQUIRED_TAG_TYPES = ("impact",)
+
+
+class DagValidationError(Exception):
+    pass
+
+
+def _execute_shell_cmd(cmd: str, cmd_params: Dict[str, dict] = dict()) -> str:
+    mask_string = "###"
+
+    cmd_params_log = {
+        key: val["value"] if not val["value"] else mask_string
+        for key, val in cmd_params.items()
+    }
+
+    print("Executing command: %s" % (cmd.format(**cmd_params_log)))
+
+    cmd_params_formatted = {
+        key: val["value"]
+        for key, val in cmd_params.items()
+    }
+
+    cmd_output = subprocess.run(
+        cmd.format(**cmd_params_formatted),
+        shell=True,
+        capture_output=True,
+        text=True,
+    )
+    try:
+        cmd_output.check_returncode()
+    except subprocess.CalledProcessError as _err:
+        print(cmd_output.stdout)
+        print(cmd_output.stderr)
+        raise _err
+
+    print("Command executed successfully, processing output...")
+
+    return cmd_output.stdout.strip().replace("\r", "").split("\n")
+
+
+def retrieve_existing_airflow_dags_from_db(pswd: str) -> Dict[str, str]:
```

Comment:
nit: name get_dags_and_tags_from_sql or similar to whatever you rename the func on line 70 to